### ISSUE TITLE
Provide training parameters and pretrained model for Jacquard Dataset

### ISSUE THREAD

**Issue by LIAGM** [skumra/robotic-grasping#31]
Hi,

Recently, our group want to use your method to test some robotic grasping scenario.
And we also want to **reproduce** the performance you mentioned in your paper.
But we found that you only provide the training parameter and pretrain model for **Cornell Dataset**.
When we use the **Jacquard Dataset** for training, we can not reproduce the accuracy such like 94.6% you mentioned in your paper.

So I am wondering if you can provide the training parameter and pretrained model for **Jacquard Dataset**?

Thank you!

**Comment by aaronhd** [comment]
I am also unable to reproduce the accuracy in Jacquard Dataset.  The dataset containing 54484 samples. We use 90% to train the model and others to evaluate. Batchsize=16, batch-per-epoch=3070(3070*16≈54484*90%). Other hyperparamters are same as the training of Cornell Dataset.

**Comment by skumra** [comment]
Released the trained models for Jacquard dataset in [release 0.3.0](https://github.com/skumra/robotic-grasping/releases/tag/v0.3.0). It also contains a fix for a bug that got introduced in last release, which could be a reason why you were unable to reproduce the results. Please try using the latest release or pull from master.

**Comment by LIAGM** [comment]
I also found that bug several days ago. 
But the performance still can not achieve what your paper said.
So can you provide some default parameters for your network training on Jacquard Dataset?

**Comment by skumra** [comment]
Great. Take a look at the updated example in the README. We also have a journal paper in review which will have more details on the hyperparameters along with an ablation study.

**Comment by LIAGM** [comment]
I have used your example to train on Jacquard dataset.
But the performance still can not achieve what your paper said like 94.6.
My reproduction is around 90% for validation and 85% for evaluation.
I also evaluate your pretrained model on Jacquard dataset and the performance is also around 90%.
And it is weird for the results is not consistent with the paper said.
